openqa-label-known-issues

```diff
@@ -175,7 +176,23 @@ investigate_issue() {
 }
 
 label_issue() {
-    local testurl="${1:?"Need 'testurl'"}"
+    local testurl="${1:-}"
+    if [[ -z "$testurl" ]]; then
+        echo "Usage: $0 https://openqa.suse.de/tests/<job_id>" >&2
+        exit 1
+    fi
+
+    if [[ $testurl =~ ^https?://([^/]+)/tests/([0-9]+)$ ]]; then
+        host="${host:-${BASH_REMATCH[1]}}"
+    else
+        echo "ERROR: Invalid URL format. Expected: https://<host>/tests/<job_id>" >&2
+        exit 1
+    fi
+
+    scheme="${scheme:-"https"}"
+    host_url="$scheme://$host"
+    client_args=(api --header 'User-Agent: openqa-label-known-issues (https://github.com/os-autoinst/scripts)' --host "$host_url" --retries="$retries")
+
```

Comment:
But now it is working only without the host definition and fail if run with `host=... openqa-label-known-issues test/<number>`

Reply:
I'm changing the script to work in both way...

Reply:
Changed!